### PR TITLE
First DirtySock PR

### PR DESCRIPTION
**Pull request by Jump-Suit**
Adds INITIAL support for the following games:
NFL Street
007 Everything Or Nothing
Nascar 08/9

Updated packets with missing properties:
OpUp
OnLn
Fbst
Rvup
User

Add initial support for the following packets:
Snap
Quik

### CODE REVIEW

**Comment by Jump-Suit** [comment]
Do not merge yet though.

**Comment by Jump-Suit** [comment]
This can be checked and merged if applicable.

File: SpecializedServers/MultiSocks/DirtySocks/Messages/UserIn.cs [hunk 1]
```diff
@@ -15,6 +17,7 @@ public override void Process(AbstractDirtySockServer context, DirtySockClient cl
 
             UserOut result = new()
             {
+                PERS = user.Username,
```
Comment (GitHubProUser67) [inline SpecializedServers/MultiSocks/DirtySocks/Messages/UserIn.cs:20]: Should be user.Persona

Comment (Jump-Suit) [reply SpecializedServers/MultiSocks/DirtySocks/Messages/UserIn.cs]: Done!

**Pull request merged**